packages/rs-dpp/src/data_contract/state_transition/data_contract_update_transition/validation/basic/validate_indices_are_backward_compatible.rs

```
@@ -230,8 +230,18 @@ fn get_wrongly_updated_non_unique_index<'a>(
             for property in
                 new_index_definition.properties[index_definition.properties.len()..].iter()
             {
-                if existing_schema.get_value(&property.name).is_ok() {
-                    return Some(index_definition);
+                if let Ok(indices) = existing_schema.get_value("indices") {
+                    let indices_array = indices.as_array().unwrap();
```

We don't do unwap neither. It will be panic as well. You should return an error